### ISSUE TITLE
Add support for event_type_id to market_filter

### ISSUE THREAD

**Issue by cpudney** [betcode-org/flumine#777]
It would be useful if `market_filter` supported an `event_type_id` so that the data files were filtered by event type, e.g. to filter by soccer you would use:

`  market_filter={"markets": data_files, 'market_types': ['MATCH_ODDS'], "event_type_id": [1], "listener_kwargs": {'inplay': False}},
`
Thank you.

**Comment by arch4672** [comment]
It already does support it, but it's event_type_ids (note the s on the end) and the ids in the list need to be strings not numbers, i.e. event_type_ids=["1"]

**Comment by liampauling** [comment]
*simulation

Fairly sure it doesn't but its been a long week

https://github.com/betcode-org/flumine/blob/a9cd71befc6062b52ce65ca695b50a56a2e81344/flumine/streams/streams.py#L26

**Comment by cpudney** [comment]
Yes, this is for simulation - apologies for the lack of context. Ref. https://discord.com/channels/970549994090295296/976702821032423444/1316748170058924052
